django_pgviews/view.py

```diff
@@ -95,10 +95,61 @@ def _drop_mat_view(cursor, view_name):
     cursor.execute("DROP MATERIALIZED VIEW IF EXISTS {0} CASCADE;".format(view_name))
 
 
+def _concurrent_index_name(view_name, concurrent_index):
+    return view_name + "_" + "_".join([s.strip() for s in concurrent_index.split(",")]) + "_index"
+
+
+def _create_concurrent_index(cursor, view_name, concurrent_index):
```

Comment:
I'm not sure this function will support all possible indexes eg GIN index with a custom opclass. Perhaps you could use the `create_sql` function of the Index object?

Reply:
This should work:
```python
GinIndex(
    fields=[...],
    opclasses=[...],
    name="gin_idx",
).create_sql(view, scema_editor, concurrently=True)
```

Reply:
Are those created in our mat. views? If so the code can handle it just fine (it doesn't throw any errors)

Reply:
Are you talking about `_create_concurrent_index` specifically? That index is a special case index for the concurrent refresh, since that requireds a unique index on the mat. view. All other indexes are not covered by this function, instead are created by the `connection.schema_editor().add_index(view_cls, index)` bit, which does handle custom indexes just fine, as internall the add_index calls just that `create_sql` you're referring to.

For example all of our indexes get created just fine on our mat. views

Reply:
Ah I see sorry, misread that, that's good then!